mode/factor/factor.js

```diff
@@ -22,50 +22,62 @@
       {regex: /#?!.*/, token: "comment"},
       // strings """, multiline --> state
       {regex: /"""/, token: "string", next: "string3"},
-      {regex: /"/, token: "string", next: "string"},
```

Comment:
It's actually improved on line 26 in the new version. In Factor, there are several words that define strings. One of them is just a quote ("). The others are words ending in a quote, which is reflected as /\S*?"/.